src/statsmodels/formula.jl

```diff
@@ -48,8 +48,12 @@ type ModelFrame
     contrasts::Dict{Symbol, ContrastsMatrix}
 end
 
-type ModelMatrix{T <: @compat(Union{Float32, Float64})}
-    m::Matrix{T}
+modelmatrixcontainertypes = [Matrix{Float32}, Matrix{Float64},
```

Comment:
Maybe we should accept any subtype of `Union{AbstractMatrix{Float32}, AbstractMatrix{Float32}}`? Or is there anything specific that only `Matrix` and `SparseMatrixCSC`?